src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -35,14 +35,18 @@ def load_arguments(self, _):
         c.argument('no_proxy', options_list=['--proxy-skip-range'], arg_group='Proxy', help='List of URLs/CIDRs for which proxy should not to be used.')
         c.argument('proxy_cert', options_list=['--proxy-cert', '--custom-ca-cert'], arg_group='Proxy', type=file_type, completer=FilesCompleter(), help='Path to the certificate file for proxy or custom Certificate Authority')
         c.argument('distribution', options_list=['--distribution'], help='The Kubernetes distribution which will be running on this connected cluster.', arg_type=get_enum_type(Distribution_Enum_Values))
+        c.argument('distribution_version', help='The Kubernetes distribution version of the connected cluster.')
         c.argument('infrastructure', options_list=['--infrastructure'], help='The infrastructure on which the Kubernetes cluster represented by this connected cluster will be running on.', arg_type=get_enum_type(Infrastructure_Enum_Values))
+        c.argument('azure_hybrid_benefit', help='Flag to enable/disable Azure Hybrid Benefit feature.', arg_type=get_enum_type(AHB_Enum_Values))
```

Comment:
Why don't we define it as `action='store_true'` but as an enumeration `AHB_Enum_Values`? What is the role of `NotApplicable`?

Reply:
The 'AzureHybridBenefit' feature is currently only available for selected type of kubernetes (AKS on HCI) and only those customers are allowed to toggle the value to 'True' or 'False'. For all the remaining kubernetes clusters, the value should be 'NotApplicable' since they are not eligible to claim this benefit. Hence we need to support all three values for this enum

Reply:
Got it, thanks~